### PR TITLE
add hitomipupil 

### PR DESCRIPTION
**Pull request by hitomipupil**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

**Comment by bermarte** [comment]
You shouldn't use the same text for 2 different commits, always explain in simple terms what that commit will do, if the commit will fix a previous problem (file empty), you will mention it.

File: student-bios/hitomipupil.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Hello I am Hitomi 🐈
+
+![hitomi](https://ca.slack-edge.com/T04N0HHDQCW-U065R2TRSG1-afb7ac8b1e26-512)
```
Comment (bermarte) [inline student-bios/hitomipupil.md:3]: Copy the file in a img folder inside student-bio, it's better to have no inline code of this kind.

**Review by bermarte — CHANGES_REQUESTED** [review]

**Comment by hitomipupil** [comment]
> You shouldn't use the same text for 2 different commits, always explain in simple terms what that commit will do, if the commit will fix a previous problem (file empty), you will mention it.

@bermarte I see. I will be careful from next time.

File: student-bios/hitomipupil.md (hunk 1 above)
Comment (hitomipupil) [reply student-bios/hitomipupil.md:3]: > Copy the file in a img folder inside student-bio, it's better to have no inline code of this kind.

I see, I will try it. Thank you for checking it.

**Review by samirm00 — APPROVED** [review]

**Pull request closed without merging**